cmd/ctl/pkg/convert/convert.go

```diff
@@ -68,15 +65,14 @@ to change to output destination.`))
 )
 
 var (
-	scheme             = runtime.NewScheme()
+	// Use the webhook's scheme as it already has the internal cert-manager types,
+	// and their conversion functions registered.
+	// In future we may we want to consider creating a dedicated scheme used by
+	// the ctl tool.
+	scheme             = webhook.Scheme
 	codecs             = serializer.NewCodecFactory(scheme)
 	parameterCodec     = runtime.NewParameterCodec(scheme)
 	localSchemeBuilder = runtime.SchemeBuilder{
-		cmapi.AddToScheme,
-		cmapiv1alpha3.AddToScheme,
-		cmacme.AddToScheme,
-		cmacmev1alpha3.AddToScheme,
-		cmmeta.AddToScheme,
 		whapi.AddToScheme,
 		kscheme.AddToScheme,
 		apireg.AddToScheme,
```

Comment:
We can remove pretty much all of these variables now (except `scheme`!) as I don't think they are used anymore. You can also remove the `init` function in this file too (which registers things with that scheme). 🎉

Reply:
I have removed the two definitely unused variables `codecs` and `parameterCodec`.
But some parts of `init()` and what is left of `localSchemeBuilder` is needed to register the kind "List". I don't know which parts that would be and which parts I can delete.